themes/theme-b2b/src/components/Tray.style.ts

```diff
@@ -0,0 +1,9 @@
+import { Theme } from '@marigold/components';
+
+export const Tray: Theme['components']['Tray'] = {
+  base: {
```

Comment:
If we have to add this to every theme, maybe we should add this as `__baseCSS` to the `Tray`?

Reply:
yes, so should we keep the possibility to style or remove the themeextension possibility?